### PR TITLE
[voq] [chassis] Bug fixes to get iface_namingmode tests to work on multi-asic linecards of a T2 chassis

### PR DESCRIPTION
**Pull request by sanmalho-git**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
Currently, the following test isfailing on multi-asic linecards on a T2 VoQ chassis.
- show pfc counters: This fails as the Inband and Recycle ports show in the 'show pfc counters' table while they are not expected.

#### How did you do it?
- show pfc counters:
   - Treat Inband and Recycle ports on a VoQ chassis as internal ports. Since the inband and recycle ports do not show up in 'show' commands unless we specify '-d all' option, they should be treated similar to internal ports.

#### How did you verify/test it?
Ran the test against a multi-asic linecard in a T2 VoQ chassis.

### CODE REVIEW

File: tests/iface_namingmode/test_iface_namingmode.py [hunk 1]
```diff
@@ -120,6 +120,9 @@ def setup_config_mode(ansible_adhoc, duthosts, enum_rand_one_per_hwsku_frontend_
     logger.info('Creating a guest user')
     duthost.user(name='guest', groups='sudo', state ='present', shell='/bin/bash')
     duthost.shell('echo guest:guest | sudo chpasswd')
+    # Add the guest user to the redis group such that it can execute show commands that need access to the redis database
+    # One such command is 'show acl table'
+    duthost.shell('sudo usermod -aG redis guest')
```
Comment (SuvarnaMeenakshi) [inline tests/iface_namingmode/test_iface_namingmode.py:125]: Why is this needed now and was not previously required?
Seems like this works fine on single asic but not on multi-asic.

Comment (SuvarnaMeenakshi) [reply tests/iface_namingmode/test_iface_namingmode.py:125]: show acl table command should be allowed for read-only user, this should be fixed in acl-loader.
@arlakshm fyi

**Review by SuvarnaMeenakshi — CHANGES_REQUESTED** [review]
Please modify as per comments.

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 1 above)
Comment (sanmalho-git) [reply tests/iface_namingmode/test_iface_namingmode.py:125]: Assuming acl-loader is in sonic-buildimage. Is the fix already merged. If not,  is there an ETA on it.

Comment (SuvarnaMeenakshi) [reply tests/iface_namingmode/test_iface_namingmode.py:125]: Fix is not merged in sonic-buildimage, PR will be available from @arlakshm soon.

Comment (SuvarnaMeenakshi) [reply tests/iface_namingmode/test_iface_namingmode.py:125]: @sanmalho-git This change can be removed from here to add guest uesr to redis group.
This is the PR to ensure the cli will work for RO user: https://github.com/sonic-net/sonic-utilities/pull/2525

**Comment by sanmalho-git** [comment]
I will close this PR once the sonic-utilities PR is merged into 202205

**Comment by rlhui** [comment]
> I will close this PR once the sonic-utilities PR is merged into 202205

@sanmalho-git  - which sonic-utilities PR is being referred to here?  Is this PR still needed? Thanks.

**Comment by sanmalho-git** [comment]
@rlhui  https://github.com/sonic-net/sonic-utilities/pull/2525 is the sonic-utilities PR that is required if we want to close this PR

File: tests/iface_namingmode/test_iface_namingmode.py (hunk 1 above)
Comment (SuvarnaMeenakshi) [reply tests/iface_namingmode/test_iface_namingmode.py:125]: We could remove this change from this PR and merge this PR.

We will track sonic-net/sonic-utilities#2525 this PR to handle the permission error that we see with show acl table.

Comment (sanmalho-git) [reply tests/iface_namingmode/test_iface_namingmode.py]: @SuvarnaMeenakshi  - I have removed the fix in iface_namingmode.py that will be fixed by https://github.com/sonic-net/sonic-utilities/pull/2525

**Review by SuvarnaMeenakshi — COMMENTED** [review]
lgtm

**Review by SuvarnaMeenakshi — APPROVED** [review]
lgtm
